pipeline/README.md

```diff
@@ -11,6 +7,9 @@ The created pipeline uses the best practices for infrastructure validation and h
 - destroy - This stage destroys the infrastructure created in the above stage.
 Running these four stages ensures the integrity of the terraform configurations.
 
+
+The deployed pipeline will fail on the first instance looking for an active connection to your VCS. The connection needs to be in an `Available` status before the pipeline execution can begin.
```

Comment:
should mention as pre-requisites for Pipeline?

Reply:
The connection is created by the same pipeline code . I just moved this under the #Before you start header.